start

```diff
@@ -91,7 +92,6 @@ function start() {
   echo "network root account id: $NETWORK_ROOT_ACCOUNT_ID"
 
   copy_defaults
-  validate_after_copy_defaults
```

Comment:
this is running too soon since it references PROTOCOL_VERSION, as `start` has not finished conditionally setting PROTOCOL_VERSION if it's not set and `local` network, in that case PROTOCOL_VERSION is set later after core has been started dynamically to the default version that the core runtime reports in `upgrade_local`